File: Data Structures & Algorithms/two-integer-sum-ii/submission-0.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        n = len(numbers)
        l = 0
        r = n - 1

        while l < r:
            if numbers[l] + numbers[r] == target:
                a = [l + 1, r + 1]
                return a
            if numbers[l] + numbers[r] > target:
                r -= 1
            if numbers[l] + numbers[r] < target:
                l += 1
        return [0, 0]

File: Data Structures & Algorithms/two-integer-sum/submission-0.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] arr = {0,0};
        for(int i = 0; i < nums.length; i ++){
            for(int k = i + 1; k < nums.length; k++){
                if(nums[i] + nums[k] == target){
                    arr[0] = i;
                    arr[1] = k;
                    return arr;
                }
            }
        }
        return arr;
    }
}
